group_vars/all.yml

```diff
@@ -43,6 +43,26 @@ adw:
 sync:
   version: 3.6.0
 
+supported_os:
+  RHEL:
+    versions:
+      - 8.5
+      - 8.4
+      - 8.2
+      - 7.7
+      - 7.6
+  CentOS:
+    versions:
+      - 7.9
+  Ubuntu:
+    versions:
+      - 20.04
+      - 18.04
+  SUSE:
+    versions:
+      - 15.0
+      - 12.0
```

Comment:
@alxgomz not sure we could really consider suse supported since we are not testing it

also strange that it's here given that is not even mentioned in https://docs.alfresco.com/content-services/latest/support/

Reply:
It's supported for ACS 6.2 (which we still support) but you're right e don't support this distro in the playbook. It needs to be removed.

Reply:
@Artur-Piechuta you can safely remove it also in https://github.com/Alfresco/alfresco-ansible-deployment/blob/60124982dc27417d38a1054f4957240987040e2a/docs/README.md#versioning